dev/airflow-pr

```diff
@@ -263,6 +263,23 @@ def merge_pr(pr_num, target_ref, title, body, pr_repo_desc, local):
             'continue.')
         click.pause(click.style(msg, fg='blue', bold=True))
 
+        # The user might have removed "Closes #XXXX" from the commit message
+        # so we add it back to make sure GitHub closes the PR.
+        close_msg = "Closes #{}".format(pr_num)
+        commit_msg = run_cmd('git log -1 --pretty=%B', echo_cmd=False)
+        if close_msg.lower() not in commit_msg.lower():
+            click.echo(
+                'The commit message does not contain "closes #{}". The phrase '
```

Comment:
For paranoia, would rather have:

   ...  contain "{}". The phrase ...

And use close_msg in the .format() call.